docker/rust/release/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM stablex-binary
+FROM stablex-binary-private
```

Comment:
Where does this change come from?

Reply:
In travis, I renamed the docker builds to stablex-binary-private and stablex-binary-public, depending on the solver